src/web-ui/src/public/ProductDetail.vue

```diff
@@ -1,233 +1,247 @@
 <template>
-<div class="container">
-
-  <!-- Loading Indicator -->
-  <div class="container mb-4" v-if="!product">
-    <i class="fas fa-spinner fa-spin fa-3x"></i>
-  </div>
-
-  <!-- Product Detail-->
-  <div class="row" v-if="product">
-    <div class="col-sm-12 col-md-6 col-lg-6">
-      <img :src="productImageURL" class="card-img-top" alt="...">
-    </div>
-    <div class="col-sm-12 col-md-6 col-lg-6">
-       <h5>{{ product.name }}</h5>
-       <p>{{ product.description }}</p>
-       <p>${{ product.price }}</p>
-       <span v-if="product.current_stock > 0">Items currently in stock: {{ product.current_stock }}</span>
-       <span v-else>Sorry, this item is currently out of stock</span>
-       <p>
-        <i class="fas fa-star"></i><i class="fas fa-star"></i><i class="fas fa-star"></i><i class="fas fa-star"></i><i class="fas fa-star"></i>
-       </p>       
-       <button class="btn btn-outline-primary" v-on:click="addToCart()" :disabled='addToCartDisabled'> Add to Cart </button>
```

Comment:
Looks like the PR has lost the new product stock behavior.

Reply:
Added this back with some additional rules around when to display that quantity dropdown and with what available quantities.